src/Wallets/Contract/UsersWalletTrait.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Kanvas\Packages\Wallets\Contract;
+
+use Kanvas\Packages\Wallets\Models\Wallets;
+use Phalcon\Mvc\Model\Resultset\Simple;
+use Kanvas\Packages\Wallets\PaymentMethods\PaymentMethods;
+
+/**
+ * Methods most used in Users wallet
+ */
+trait UsersWalletTrait
+{
+    public ?PaymentMethods $payment_methods = null;
+
+    /**
+     * We get all the user's wallets
+     * @return Simple
+     */
+    public function getWallets() : ?Simple
```

Comment:
@dasdo it should be a ResultsetInterface instead of simple https://github.com/bakaphp/baka/blob/0.6/src/Database/Model.php#L159